include/ocvsmd/sdk/defines.hpp

```diff
@@ -115,6 +122,18 @@ enum class CyphalPriority : std::uint8_t
 
 };  // CyphalPriority
 
+/// Defines helper which owns mutable raw data buffer.
+///
+struct OwnMutablePayload
```

Comment:
```suggestion
struct OwnedMutablePayload
```

?

Reply:
Will fix it in the next pr (where I'm adding a bit more unit tests)